changelog.d/gh-8037.fixed

```diff
@@ -0,0 +1,13 @@
+Constant propagation is now applied to declarations inside stack array declarations in C; so
```

Comment:
I believe you wanted to write "... is now applied inside stack array ..." and the "to declarations" that is there in the middle may be the result of you having rewritten this sentence?

Reply:
Ah Indeed; removed the `to declarations inside`.